ArduinoCode/Driver-IO/RTDS/RTDS/RTDS.ino

```diff
@@ -0,0 +1,68 @@
+//By Joshua Cheng
+//Oct 12, 2020
+
+/* detects shutdown voltage, if voltage is OK (12v scaled down) allow button (turn on button LED) 
+ *  to turn speaker on for three seconds 
+ */
```

Comment:
Clean comments should have proper indentation, spacing, and capitalization for consistency. Be more descriptive if it is appropriate to enumerate the possible functions of this code. Comments are also generally attached to specific functions, and a comment for the entire file should be generalized rather than specific.